x/leverage/keeper/liquidate.go

```diff
@@ -210,15 +216,24 @@ func ComputeCloseFactor(
 		return sdk.OneDec()
 	}
 
-	// outside of special cases, close factor scales linearly between MinimumCloseFactor and 1.0,
-	// reaching max value when (borrowed / threshold) = 1 + CompleteLiquidationThreshold
+	if collateralValue.LTE(liquidationThreshold) {
+		// If all tokens have a collateral weight of 1, there is no room for dynamic close factor
+		return sdk.OneDec()
+	}
+
+	// critical borrowed value = liquidationThreshold + CLT * (collateralValue - liquidationThreshold)
```

Comment:
```suggestion
```

Reply:
I've added a complete documentation to the function doc string, so this line is not needed.